doc/rdoc/markup_reference.rb

```diff
@@ -169,6 +173,49 @@
 # Pro tip:  If your indented Ruby code does not get highlighted,
 # it may contain a syntax error.
 #
+# ==== Block Quotes
+#
+# You can use the characters <tt>>>></tt> (unindented),
+# followed by indented text, to treat the text
+# as a {block quote}[https://en.wikipedia.org/wiki/Block_quotation]:
+#
+# Example input:
+#
+#   >>>
```

Comment:
The `>>>` here doesn't seem to correctly generate for me. I'm not sure what the best solution to this is.

<img width="979" alt="Screen Shot 2022-07-28 at 1 31 33 PM" src="https://user-images.githubusercontent.com/15860699/181600945-b13e8fac-735d-4f44-8781-2993cb3de858.png">

Reply:
I got a period, too.  I tried escaping one or all three characters; didn't work.  May be a bug.

Reply:
I think fixed it now.
Can you rebase this PR?

Reply:
Thanks, @nobu, for the fix.  I don't know rebase.  Maybe @peterzhu2118 can help?

Reply:
I did a rebase. The bug is fixed. Thanks @nobu!